### PR TITLE
Added an option for making punishments silent with plugin messaging.

### PR DESCRIPTION
**Pull request by MelonCola**
Requested by a user of my ban GUI plugin.

### CODE REVIEW

**Comment by SpooderCraft** [comment]
Yes we need that

**Comment by DevLeoko** [comment]
This would cause NPEs if there is no silent argument

File: src/main/java/me/leoko/advancedban/bungee/listener/InternalListener.java [hunk 1]
```diff
@@ -62,7 +62,7 @@ public void onPluginMessageEvent(PluginMessageEvent e) {
                             TimeManager.getTime() + punishment.get("end").getAsLong(),
                             punishment.get("calculation") != null ? punishment.get("calculation").getAsString() : null,
                             -1
-                    ).create();
+                    ).create(punishment.get("silent").getAsBoolean());
```
Comment (DevLeoko) [inline src/main/java/me/leoko/advancedban/bungee/listener/InternalListener.java]: Add a not-null check right here

**Comment by MelonCola** [comment]
Edited.

**Pull request merged**